set-web/src/main/java/eu/europeana/set/web/service/controller/jsonld/WebUserSetRest.java

```diff
@@ -642,86 +596,45 @@ private void validateUpdateItemsByProfile(UserSet storedUserSet, UserSet updateU
 	@RequestMapping(value = {"/set/{identifier}"}, method = RequestMethod.DELETE)
 	@ApiOperation(value = "Delete an existing user set", nickname = "delete", response = java.lang.Void.class)
 	public ResponseEntity<String> deleteUserSet(
-			@RequestParam(value = WebUserSetFields.PARAM_WSKEY, required = false) String apiKey,
 			@PathVariable(value = WebUserSetFields.PATH_PARAM_SET_ID) String identifier,
-			@RequestParam(value = WebUserSetFields.USER_TOKEN, required = false, defaultValue = WebUserSetFields.USER_ANONYMOUNS) String userToken,
 			HttpServletRequest request
 			) throws HttpException {
 
-		userToken = getUserToken(userToken, request);
-				
-		return deleteUserSet(request, identifier, apiKey, userToken);
+		return deleteUserSet(request, identifier);
```

Comment:
the verifyWriteAccess method needs to be invoked here

Reply:
verifyWriteAccess invoked as suggested